config/software/libzmq.rb

```diff
@@ -18,7 +18,7 @@
 name "libzmq"
 default_version "2.1.11"
 
-license "GPL-3.0"
+license "GPL-3.0 (with exception)"
```

Comment:
Is this a valid license from Omnibus's perspective @sersut? Or does it have to appear in this list:
https://github.com/chef/omnibus/blob/master/lib/omnibus/licensing.rb#L302